test/e2e/service.go

```diff
@@ -1092,6 +1113,14 @@ spec:
 
 		_, err = curlInContainer(clientContainer, svcIP, endpointHTTPPort, "big.iso -o big.iso", 120)
 		framework.ExpectNoError(err, "failed to curl load balancer service")
+
+		ginkgo.By("rest MTU on intermediary router to allow large packets")
```

Comment:
i think you meant "reset" here. Regardless, this should not be a block that executes conditionally on the previous steps not failing. It needs to be in a defer or an after each so that if this test case fails, we still reset the MTU.

Reply:
yes "reset"

hmm ok, only if the 1200  command succeeded we technically need to set this back to 1500, but I can do this in a defer.